From ffce6863cb0470992612394a16a9f794bc4937be Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Sun, 29 Jan 2023 22:35:34 +0200
Subject: [PATCH] [SW-120977] ppboot: rename bis mode chk api

Rename preboot_check_bis_compliant_mode to
preboot_is_bis_compliant_mode, which is a more
suitable name to a function which returns boolean.

Change-Id: Ic7854a8dde0549873d67aab04c33b23f7ff8c406
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/app/ext_rom/erom_pcie.c b/app/ext_rom/erom_pcie.c
index 0296eea..3f965fa 100644
--- a/app/ext_rom/erom_pcie.c
+++ b/app/ext_rom/erom_pcie.c
@@ -231,7 +231,7 @@
  * here as a strong function with an empty implementation, and
  * in security module the function is declared as a weak function.
  */
-bool preboot_check_bis_compliant_mode(void)
+bool preboot_is_bis_compliant_mode(void)
 {
 	return true;
 }
diff --git a/arch/gaudi2/arc/common.c b/arch/gaudi2/arc/common.c
index b4ebc33..cd2247c 100644
--- a/arch/gaudi2/arc/common.c
+++ b/arch/gaudi2/arc/common.c
@@ -893,7 +893,7 @@
 	 * On HL225B cards, no debug features are allowed.
 	 * Enabling RMA on HL225B card, should not disable security
 	 */
-	if (preboot_check_bis_compliant_mode())
+	if (preboot_is_bis_compliant_mode())
 		return false;
 
 	module_id_3 = gaudi2_gpio_read(GAUDI2_MOD_ID_3_GPIO);
diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index 8b924e2..4806b0b 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -528,7 +528,7 @@
 		      mmPCIE_DBI_SUBSYSTEM_ID_SUBSYSTEM_VENDOR_ID_REG);
 
 	/* Update REV ID for HL225B */
-	if (preboot_check_bis_compliant_mode()) {
+	if (preboot_is_bis_compliant_mode()) {
 		class_rev = regmap_readl(mmPCIE_DBI_CLASS_CODE_REVISION_ID);
 
 		/* clear low byte in PCIE class low word */
diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index 1bc2205..e7529a2 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -1507,7 +1507,7 @@
 	reset_arc_farm_arc0();
 }
 
-__weak bool preboot_check_bis_compliant_mode(void)
+__weak bool preboot_is_bis_compliant_mode(void)
 {
 	u32 config_fuse;
 	int ret = preboot_fuse_read(EFUSE_SYSCFG_OFFSET, &config_fuse);
diff --git a/include/preboot.h b/include/preboot.h
index 508ff3e..798a0f1 100644
--- a/include/preboot.h
+++ b/include/preboot.h
@@ -200,7 +200,7 @@
 void preboot_dynamic_img_elbi_reg_cleanup(void *map_addr, u32 size);
 bool preboot_is_overlap_address_range(u64 addr, int len);
 
-bool preboot_check_bis_compliant_mode(void);
+bool preboot_is_bis_compliant_mode(void);
 enum hl_fw_en_caps;
 
 void preboot_dev_enable_caps(void);
